### PR TITLE
Seungin/east hall proto

### PR DESCRIPTION
**Pull request by SeunginLyu**
Followed Olin Library style guide, tweaked markdown render css, added routes for different views of Futureboard

### CODE REVIEW

File: static/js/board.js [hunk 1]
```diff
@@ -1,117 +1,9 @@
 /*
 Main Futureboard general JS components :
-Contains socket configuration, listens on board controls,
-Executes functions when document is ready(auto scroll, carousel interval).
 */
 
 // Socket.io configuration
 var socket = io();
```
Comment (aidankmcl) [inline static/js/board.js]: This file got pretty skinny, is it being included separately to make it more explicit that other files are using sockets? May not be too bad to toss this into the html template (more clear that it's specifically for other files to use)

File: static/css/slideshow-featured-events.css [hunk 2]
```diff
@@ -195,18 +262,29 @@ div.image-holder {
...
+  height:35vh;
+  margin-left:3vh;
+  margin-right:3vh;
+  float:left;
+}
+
+/*//hide scroll bar*/
+::-webkit-scrollbar {
+    width: 0px;
+    background: transparent; /* make scrollbar transparent */
```
Comment (aidankmcl) [inline static/css/slideshow-featured-events.css:289]: I'd be curious about whether people like scrollbar or not (to show how many more events are upcoming)

File: static/js/board_events.js [hunk 3]
```diff
@@ -12,8 +12,7 @@ function populateEvents(eventsData) {
     let eventStart = Date.parse(eventsData[i].start).toString("YYMMdd");
     let today = Date.today().toString("YYMMdd");
     // let tomorrow = (1).day().fromNow().toString("MMdd");
-    // let thisWeek = (7).day().fromNow().toString("MMdd");
-
+    // let thisWeek = (7).day().fromNow().toString("MMdd");
```
Comment (aidankmcl) [inline static/js/board_events.js]: remove commented

File: static/js/board_events.js [hunk 4]
```diff
@@ -32,13 +31,15 @@ function populateEvents(eventsData) {
   * @param {Object} eventData - A single JSON instance of ABE event
 */
 function createEventObject(eventData) {
+  var converter = new showdown.Converter(); // markdown converter
```
Comment (aidankmcl) [inline static/js/board_events.js:32]: sweeeeet!

**Review by aidankmcl — COMMENTED** [review]
lookin good! A couple pieces here and there but generally very excited

File: static/css/slideshow-featured-events.css (hunk 2 above)
Comment (SeunginLyu) [reply static/css/slideshow-featured-events.css:289]: I'll add this as an issue

File: static/js/board.js (hunk 1 above)
Comment (SeunginLyu) [reply static/js/board.js]: renamed board.js -> init_socket.js

File: static/css/slideshow-featured-events.css (hunk 2 above)
Comment (SeunginLyu) [reply static/css/slideshow-featured-events.css:289]: #46

**Pull request merged**